README.md

```diff
@@ -88,7 +104,8 @@ In [prod.tfvars](https://github.com/hmcts/azure-enterprise/blob/main/environment
 
 ```terraform
 cft_non_production_subscriptions = {
-  DCD-CFTAPPS-DEV = {}
+  DCD-CFTAPPS-DEV = {
```

Comment:
I would probably keep these all on one line if they don't have customisations, I think that's what `terraform fmt` will do. (Applies throughout)